### DISCUSSION TITLE
十一月更新 - 3

### DISCUSSION THREAD

**Discussion by Richasy** [Richasy/Bili.Uwp#669] [通知]
## 🚀 功能

- 支持检查上次应用关闭时正在播放的视频 @Richasy (#666)

## 🐛 Bug 修复

- 修复多P视频历史记录没有正确上传的问题 @Richasy (#667)
- 修复账户异常导致循环登录的问题 @Richasy (#663)
- 优化单击和切换分集的行为 @Richasy (#662)
- 修复PGC内容部分字段解析失败的问题 @Richasy (#661)
- 修复视频播放速率对直播造成的影响 @Richasy (#660)

<hr /><em>This discussion was created from the release <a href='https://github.com/Richasy/Bili.Uwp/releases/tag/v2.2111.3.0'>十一月更新 - 3</a>.</em>

**Comment by xyl23333** [comment]
前排

**Reply by zixulu** [reply]
前几天才发现GitHub有新版，然后就更新了三次hhh

**Reply by oopstls** [reply]
越来越好用了 哈哈哈

**Comment by nightmr** [comment]
啥时候可以支持hdr呢？还是hdr在Windows上就是无解了？

**Reply by hez2010** [reply]
Windows 是自带 HDR 支持的，可以在系统设置中看看。

**Comment by nightmr** [comment]
我是说在Windows下没办法播放b站的hdr视频。换老edge那个办法问题多多，而且很不方便。

**Comment by YanSu5001** [comment]
播放器能不能不要鼠标点击就暂停啊！因为播放器的控制条不能自己消失，只能点一下鼠标才能消失，这样的话就需要点两下才能行，非常的麻烦。

**Comment by microLANG** [comment]
![image](https://user-images.githubusercontent.com/49826042/152669715-cc8e88bf-f9b8-45b0-b057-39c03125c52a.png)
此应用包未使用受信任的证书签名。请与系统管理员或应用程序开发人员联系，以获取新证书或带有受信任证书的应用程序包。应用程序包中签名的根证书和所有即时证书必须受信任（0x800B010A） 
**_怎么办？_**

**Reply by YanSu5001** [reply]
[Windows 10如何安装APPX.pdf](https://github.com/Richasy/Bili.Uwp/files/8009628/Windows.10.APPX.pdf)

**Reply by YanSu5001** [reply]
证书没有安装或者安装位置错误。按照上一条评论的文件的方法安装即可
